examples/asr/speech2text.py

```diff
@@ -59,9 +59,9 @@ def main():
     parser.add_argument("--wandb_exp_name", default=None, type=str)
     parser.add_argument("--wandb_project", default=None, type=str)
     parser.add_argument("--max_train_audio_len", default=16.7, type=float, help="max audio length")
-    parser.add_argument("--trim_silence", default=True, type=bool, help="trim audio from silence or not")
+    parser.add_argument("--trim_silence", default=True, type=eval, help="trim audio from silence or not")
```

Comment:
Thanks for catching this. Can we switch both to `parser.add_argument("--do_not_trim_silence", type="store_false", help="Add this flag to disable silence trimming")` and `parser.add_argument("--do_not_normalize_text", type="store_false", help="Add this flag to set to False for non-English.")`